tests/common.go

```diff
@@ -3939,6 +3956,30 @@ func CreateCloudCredential(provider, credName string, uid, orgID string, ctx con
 				},
 			},
 		}
+
+	case drivers.ProviderIbm:
+		log.Infof("Create creds for IBM")
+		// PA-1328
+		apiKey, err := GetIBMApiKey("default")
```

Comment:
This is fixed what are we trying to say with the comment here?

Reply:
Done

Reply:
Removed it